packages/react-ui/src/widgets/CategoryWidgetUI.js

```diff
@@ -271,11 +271,13 @@ function CategoryWidgetUI(props) {
         return searchValue
           ? list.filter((elem) => {
               return (
-                elem.name.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1 ||
-                (labels[elem.name]
-                  ? labels[elem.name].toLowerCase().indexOf(searchValue.toLowerCase()) !==
-                    -1
-                  : false)
+                elem.name !== null &&
+                (elem.name?.toLowerCase().indexOf(searchValue.toLowerCase()) !== -1 ||
```

Comment:
I would include also `!== undefined` and by that way you can avoid the optional chaining after name.

Reply:
Done! 😄